daisy/step_wait_for_instances_signal.go

```diff
@@ -83,8 +104,8 @@ func waitForSerialOutput(s *Step, project, zone, name string, so *SerialOutput,
 	if so.SuccessMatch != "" {
 		msg += fmt.Sprintf(", SuccessMatch: %q", so.SuccessMatch)
 	}
-	if so.FailureMatch != "" {
-		msg += fmt.Sprintf(", FailureMatch: %q", so.FailureMatch)
+	if len(so.FailureMatch) > 0 {
+		msg += fmt.Sprintf(", FailureMatch: %v", so.FailureMatch)
```

Comment:
Keep this to %q, it'll quote each element in the list

Reply:
Done